configs/setups/awicm3/awicm3.yaml

```diff
@@ -299,15 +317,52 @@ fesom:
         choose_general.version:
                 "v3.0":
                         namelist_dir: "${esm_namelist_dir}/fesom2/2.0/awicm3/v3.0/"
+                        add_namelist_changes:
+                                namelist.oce:
+                                        oce_tra:
+                                                surf_relax_s: "${surf_relax_s}"
                 "v3.1":
                         namelist_dir: "${esm_namelist_dir}/fesom2/2.0/awicm3/v3.1/"
+                        add_namelist_changes:
+                                namelist.oce:
+                                        oce_tra:
+                                                surf_relax_s: "${surf_relax_s}"
                 "master":
                         namelist_dir: "${model_dir}/config/"
-                "frontiers":
-                        namelist_dir: "${esm_namelist_dir}/fesom2/2.0/awicm3/frontiers/"
+                        add_namelists:
+                                - namelist.tra
+                                - namelist.cvmix
+                        add_config_sources:
+                                tra:     "${namelist_dir}/namelist.tra"
+                                dyn:     "${namelist_dir}/namelist.dyn"
+                        add_config_in_work:
+                                tra:     "namelist.tra"
+                                dyn:     "namelist.dyn"
```

Comment:
I would recommend to add these outside of the choose block, in fact the best would be to add them to the `fesom-2.1.yaml` if we would be using that file at all, but give that that's now the case. `sources` and `in_work` are not used unless `files` points at their tags.

Reply:
wont that send warnings about missing namelist templates to users of v3.0 and v3.1?

Reply:
It would, you could consider adding them to the "allowed to be missing" list. 

Have a look here for the code: https://github.com/esm-tools/esm_tools/blob/f94c5dfc7b70c884e39526927ab6d33eaf95a408/src/esm_runscripts/filelists.py#L1006

And here for example tests: https://github.com/esm-tools/esm_tools/blob/f94c5dfc7b70c884e39526927ab6d33eaf95a408/tests/test_esm_runscripts/test_filelists.py#L6